mllib/src/main/scala/org/apache/spark/mllib/recommendation/MatrixFactorizationModel.scala

```diff
@@ -103,13 +109,106 @@ class MatrixFactorizationModel private[mllib] (
     recommend(productFeatures.lookup(product).head, userFeatures, num)
       .map(t => Rating(t._1, product, t._2))
 
+  /**
+   * Recommends topK users/products.
+   *
+   * @param num how many users to return. The number returned may be less than this.
```

Comment:
This block doesn't associate with any method. Please move these docs to the corresponding methods' JavaDoc.